v1/suzu.json

```diff
@@ -77,6 +91,23 @@
           "to_state": "bootloader",
           "fallback_user_action": "bootloader"
         },
+        {
+          "type": "fastboot:flash",
+          "condition": {"var": "bootstrap", "value": true},
+          "flash": [
+            {
+              "partition": "oem",
+              "file": "SW_binaries_for_Xperia_AOSP_N_MR1_5.7_r1_v08_loire.img",
+              "group": "firmware",
+              "raw": true
```

Comment:
didn't you say you did not need raw?

Reply:
D'oh. Wrong copy open in my editor. Updating now. Do I cancel this PR and create a new one?

Reply:
Applied correct changes to my branch in this PR.